tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/IClusterNodeIntegration.cs

```diff
@@ -0,0 +1,86 @@
+// <copyright file="IClusterNodeIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Couchbase
+{
+    /// <summary>
+    /// Couchbase IclusterNode 3.2 calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+       AssemblyName = CouchbaseCommon.CouchbaseClientAssemblyName,
+       TypeName = "Couchbase.Core.ClusterNode",
+       MethodName = "ExecuteOp",
+       ReturnTypeName = ClrNames.Task,
+       ParameterTypeNames = new[] { CouchbaseCommon.CouchbaseOperationV3TypeName, "Couchbase.Core.IO.Operations.CancellationTokenPair" },
+       MinimumVersion = "3.1.3",
+       MaximumVersion = CouchbaseCommon.MaxVersion3,
+       IntegrationName = CouchbaseCommon.IntegrationName)]
+    [InstrumentMethod(
+       AssemblyName = CouchbaseCommon.CouchbaseClientAssemblyName,
+       TypeName = "Couchbase.Core.ClusterNode",
+       MethodName = "SendAsync",
+       ReturnTypeName = ClrNames.Task,
+       ParameterTypeNames = new[] { CouchbaseCommon.CouchbaseOperationV3TypeName, "Couchbase.Core.IO.Operations.CancellationTokenPair" },
+       MinimumVersion = "3.1.3",
+       MaximumVersion = CouchbaseCommon.MaxVersion3,
+       IntegrationName = CouchbaseCommon.IntegrationName)]
+    [InstrumentMethod(
+       AssemblyName = CouchbaseCommon.CouchbaseClientAssemblyName,
+       TypeName = "Couchbase.Core.ClusterNode",
+       MethodName = "ExecuteOp",
+       ReturnTypeName = ClrNames.Task,
+       ParameterTypeNames = new[] { CouchbaseCommon.CouchbaseOperationV3TypeName, ClrNames.CancellationToken },
+       MinimumVersion = "3.0.7",
+       MaximumVersion = "3.1.2",
+       IntegrationName = CouchbaseCommon.IntegrationName)]
+    [InstrumentMethod(
+       AssemblyName = CouchbaseCommon.CouchbaseClientAssemblyName,
+       TypeName = "Couchbase.Core.ClusterNode",
+       MethodName = "SendAsync",
+       ReturnTypeName = ClrNames.Task,
+       ParameterTypeNames = new[] { CouchbaseCommon.CouchbaseOperationV3TypeName, ClrNames.CancellationToken },
+       MinimumVersion = "3.0.7",
+       MaximumVersion = "3.1.2",
+       IntegrationName = CouchbaseCommon.IntegrationName)]
+    // ReSharper disable once InconsistentNaming
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class IClusterNodeIntegration
```

Comment:
Should this class (and file) be called `ClusterNodeIntegration`? (without the `I` prefix)

Reply:
Same comment for other types/files names `IClusterNode*`.

Reply:
Makes sense. I used the same format as the v2 instrumentation even though here there's only one implementation. Will fix